kotlin/src/test/kotlin/com/iamdorra/gameoflife/GridManagerTest.kt

```diff
@@ -0,0 +1,101 @@
+package com.iamdorra.gameoflife
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+internal class GridManagerTest {
+    @Test
+    fun  `should not change cells if all cells are dead`()
+    {
+        val gridManager = GridManager()
+        val grid = mutableListOf(mutableListOf<Cell>());
```

Comment:
Je suis chaud pour leur mettre un 
```suggestion
       var grid =  arrayOf(arrayOf(Cell(), Cell()), arrayOf(Cell(), Cell())) 
```

Reply:
J'ai utilisé l'initializer à la place